public/static/docs/command-reference/init.md

```diff
@@ -1,49 +1,178 @@
 # init
 
-This command initializes a <abbr>DVC project</abbr> on a directory.
-
-Note that by default the current working directory is expected to contain a Git
-repository, unless the `--no-scm` option is used.
+This command initializes a <abbr>DVC project</abbr> in a directory.
 
 ## Synopsis
 
 ```usage
-usage: dvc init [-h] [-q | -v] [--no-scm] [-f]
+usage: dvc init [-h] [-q | -v] [--no-scm] [-f] [--subdir]
 ```
 
 ## Description
 
+In its default mode, DVC works on top of a Git repository for all features to be
+available and to provide most of its value. It means that `dvc init` expects Git
+to be initialized (usually means that `.git` directory is present) in the
+directory you run `dvc init` in. Another way to put it, by default it expects to
+be run in the Git repository root.
+
+One of the explicit options can be used to set an advanced workflow in cases
+like monorepo, automation or other relevant scenarios - see:
+
+- [Initializing DVC in subdirectories](#initializing-dvc-in-subdirectories) -
+  e.g. to deal with monorepo and nested DVC projects, etc.
+- [Initializing DVC without Git](#how-does-it-affect-dvc-commands) - e.g. when
+  you need to use other than Git SCM, or some production automation case, etc.
+
 After DVC initialization, a new directory `.dvc/` will be created with the
-`config` and `.gitignore` files. These and other files and directories are
-hidden from user, as typically there's no need to interact with them directly.
-See [DVC Files and Directories](/doc/user-guide/dvc-files-and-directories) to
-learn more.
+`.dvc/config`, `.dvc/.gitignore` and other files. These files and directories
+are hidden from the user (but committed with Git), as typically there's no need
+to interact with them directly. See
+[DVC Files and Directories](/doc/user-guide/dvc-files-and-directories) to learn
+more about the structure of the `.dvc/` directory.
```

Comment:
I'm going to revert this because there's no significant difference and it's a paragraph that's repeated in several docs.

Reply:
yeah, not a big deal ... I just didn't like `... and other` and other directories - not cleat what other?  too complicated - I just tried to make it very simple

Reply:
I'll revise it in all the docs where this exists.

Reply:
👍

Reply:
Done in bc43303c for #1014